apps/src/applab/designMode.js

```diff
@@ -547,6 +548,24 @@ function duplicateScreen(element) {
   return newScreen;
 }
 
+designMode.onCopyElementToScreen = function (element, destScreen) {
+  const sourceScreen = $(element);
+  designMode.changeScreen(destScreen);
+
+  // Unwrap the draggable wrappers around the elements in the source screen:
+  const madeUndraggable = makeUndraggable(sourceScreen.children());
+
+  let duplicateElement = $(element).clone(true)[0];
```

Comment:
Should we reuse `sourceScreen` (or however we rename it) instead of re-wrapping `element` with jQuery here?

Reply:
Done.